prac_05/word_occurrences.py

```diff
@@ -0,0 +1,24 @@
+"""
+CP1404 Practical
+Count the number of words in a string
+"""
+
+words_dict = {}
```

Comment:
"dict" can be assumed by the syntax of {}.  What will the value be for each key?